pynuodb/encodedsession.py

```diff
@@ -188,6 +188,13 @@ def check_auth(self):
         except SessionException as e:
             raise ProgrammingError('Failed to authenticate: ' + str(e))
 
+    def get_auth_types(self):
```

Comment:
Why count on a trip to the TE when a protocol check will tell you what you want to know?